compiler/cpp/src/thrift/generate/t_erl_generator.cc

```diff
@@ -715,21 +755,30 @@ string t_erl_generator::render_default_value(t_field* field) {
       return "dict:new()";
     }
   } else if (type->is_set()) {
-    return "sets:new()";
+    return render_default_sets_value();
   } else if (type->is_list()) {
     return "[]";
   } else {
     return "undefined";
   }
 }
 
+string t_erl_generator::render_default_sets_value() {
+  switch (sets_to_) {
+  case SetsTo::V1:
+    return "sets:new()";
+  case SetsTo::V2:
+    return "sets:new([{version,2}])";
+  }
+}
```

Comment:
there's something missing obviosuly

Reply:
I've added default branch to switch here